core/src/main/java/com/adobe/aem/guides/spascreens/core/models/ImageMaps.java

```diff
@@ -0,0 +1,15 @@
+package com.adobe.aem.guides.spascreens.core.models;
+
+public interface ImageMaps {
```

Comment:
Can we rename this to Hotspot? It's going to be confusing to refer to image maps when talking about hotspots